db/seeds.rb

```diff
@@ -7,6 +7,10 @@
 Dir.glob(Rails.root.join('db/seeds/**/*.yml')).each do |f|
   basename = Pathname.new(f).relative_path_from(Pathname.new(Rails.root.join('db/seeds'))).to_s
   type = basename.gsub('.yml', '').singularize.classify.constantize
-  data = YAML.load_file f
-  type.create data
+  begin
+    data = YAML.load_file f
+    type.create data
+  rescue StandardError => e
+    puts "Got error #{e}. Continuing..."
```

Comment:
What error were you getting that made this necessary? It's good to have it, but if there's a bug we should also fix that.

Reply:
Huh?